genie-web/src/main/resources/static/scripts/Job.js

```diff
@@ -77,19 +77,19 @@ export default class Job extends Page {
         value: "desc",
         type: "sortOption",
         optionValues: ["desc", "asc"]
-      }
-    ];
-  }
-
-  get hiddenFormFields() {
-    return [
+      },
       { label: "Tags", name: "tag", value: "", type: "input" },
       { label: "Cluster Name", name: "clusterName", value: "", type: "input" },
       { label: "Cluster ID", name: "clusterId", value: "", type: "input" },
       { label: "Command Name", name: "commandName", value: "", type: "input" },
       { label: "Command ID", name: "commandId", value: "", type: "input" },
       { label: "Grouping", name: "grouping", value: "", type: "input" },
-      { label: "Grouping Instance", name: "groupingInstance", value: "", type: "input" },
+      {
```

Comment:
Any reason for changing this?

Reply:
Its the code formatter. We are using [Prettier](https://prettier.io/) and its the default formatting for a complex object.

Reply:
It decided to format that one line while leaving all the other lines above untouched? :-/

Reply:
That's cause Prettier defaults to line width of 80 and the last object if written on one line has a length of 87. And All other lines are under 79.